proton

```diff
@@ -1370,9 +1427,11 @@ class Session:
 
         # CoD: Black Ops 3 workaround
         if os.environ.get("SteamGameId", 0) == "311210":
-            rc = self.run_proc([g_proton.wine_bin, "c:\\Program Files (x86)\\Steam\\steam.exe"] + sys.argv[2:] + self.cmdlineappend)
+            argv = [g_proton.wine_bin, "c:\\Program Files (x86)\\Steam\\steam.exe"]
         else:
-            rc = self.run_proc([g_proton.wine64_bin, "c:\\windows\\system32\\steam.exe"] + sys.argv[2:] + self.cmdlineappend)
+            argv = [g_proton.wine64_bin, "c:\\windows\\system32\\steam.exe"]
+
+        rc = self.run_proc(adverb + argv + sys.argv[2:] + self.cmdlineappend)
```

Comment:
Not really related to this feature at all, but if we *don't* have a remote debug process, it might be nicer if the `proton` script did an `execve()` to completely replace itself with the `wine64 ...` command?

Reply:
I'll have a look into this. Sounds sensible, but I wonder if there's a reason to have it like that. Only thing that comes to mind is to have proton easily identifiable in the process tree.